test/mail_ssl_letsencrypt.bats

```diff
@@ -241,7 +241,6 @@ function _should_extract_on_changes() {
 
   # Expected log lines from the changedetector service:
   run $(_get_service_logs 'changedetector')
-  assert_output --partial 'Change detected'
```

Comment:
Just curious about motivation for removal? Doesn't provide any value I assume?

Reply:
Commit message for 375d30e7e4bafa485787999a101e44cbc823cb07 says:

``` TXT
...
Moreover,

assert_output --partial 'Change detected'

was dropped because it appears first in the logs, and due to the now
more verbose log, we'd need to increase the lenght of the `tail`ed
output, but this an unreliable method. I therefore simply dropped this
message, and I think this is fine, because the other messages are still
fine for testing whether all is well with the script and its
functionality.
```